tasks/Debian.yml

```diff
@@ -52,6 +55,7 @@
   when:
     - ansible_version.full is version_compare('2.4', '>=')
     - ansible_selinux.status == "enabled"
+  tags: install
```

Comment:
configure

Reply:
Technically this is part of the install process, we mirror the package install temporarily, while it is not available on Debian. No specific configuration is done in this step.